CHANGELOG.md

```diff
@@ -7,17 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [[Unreleased]]
 
-### Fixed
-- Added `MPTCurrency` support in `Issue` (rippled internal type)
-- Fixed the implementation error in get_latest_open_ledger_sequence method. The change uses the "current" ledger for extracting sequence number.
-- Increase default maximum payload size for websocket client
-- Fixed the default behavior of flags field when preparing transactions. By default, flags are not part of the transaction if not explicitly provided.
-- Added support for `amm_info` to `Request.from_dict`
-- Improved error handling for `amm_info`
-
 ### Added
 - Improved validation for models to also check param types
 - Support for `Account Permission` and `Account Permission Delegation` (XLS-74d, XLS-75d)
+- Support for the `Batch` amendment (XLS-56d)
+
+### Fixed
+- Add `MPTCurrency` support in `Issue` (rippled internal type)
+- Fix the implementation error in get_latest_open_ledger_sequence method. The change uses the "current" ledger for extracting sequence number
+- Increase default maximum payload size for websocket client
+- Fix the default behavior of flags field when preparing transactions. By default, flags are not part of the transaction if not explicitly provided
+- Add support for `amm_info` to `Request.from_dict`
+- Improve error handling for `amm_info`
+- Handle autofilling better when multisigning transactions
+- Improve typing for transaction-related helper functions
+- Improve handling of `TicketSequence`
+- Fix issue with failing on a higher than expected fee
```

Comment:
if you can explain the context of this CL entry, that will be helpful.

Reply:
https://github.com/XRPLF/xrpl-py/commit/341d91a7bf2f51cb040cfb34ae639df9ce980b2c

Essentially, the `max` operator wasn't working properly since it was comparing strings, not numbers.